modules/tensor_mechanics/src/materials/RecomputeRadialReturn.C

```diff
@@ -140,6 +140,19 @@ RecomputeRadialReturn::getIsotropicShearModulus()
   return shear_modulus;
 }
 
+Real
+RecomputeRadialReturn::getIsotropicBulkModulus()
+{
+  Real shear_modulus = getIsotropicShearModulus();
+  // dilatational modulus is defined as lambda plus two mu
+  Real dilatational_modulus = _elasticity_tensor[_qp](0,0,0,0);
+  if (_mesh.dimension() == 3 && dilatational_modulus != _elasticity_tensor[_qp](2,2,2,2))
+    mooseError("Check to ensure that your Elasticity Tensor is truly Isotropic");
+  Real lambda = dilatational_modulus - 2.0 * shear_modulus;
+  Real bulk_modulus = lambda + 2.0 * shear_modulus / 3.0;
```

Comment:
I suggest making all symbols that are only ever assigned once at declaration time `const`.